eng/Versions.props

```diff
@@ -143,7 +143,7 @@
     <MoqVersion>4.12.0</MoqVersion>
     <FsCheckVersion>2.14.3</FsCheckVersion>
     <!-- Docs -->
-    <MicrosoftPrivateIntellisenseVersion>3.0.0-preview-20200715.1</MicrosoftPrivateIntellisenseVersion>
+    <MicrosoftPrivateIntellisenseVersion>5.0.0-preview-20200914.1</MicrosoftPrivateIntellisenseVersion>
```

Comment:
```suggestion
    <MicrosoftPrivateIntellisenseVersion>5.0.0-preview-20200924.1</MicrosoftPrivateIntellisenseVersion>
```

Reply:
Oh but I think this RC2 change needs to be updated via the bot command in the master change. Don't apply the suggestion.